.github/workflows/gn_examples.yaml

```diff
@@ -33,12 +33,20 @@ jobs:
               run: scripts/examples/gn_nrf_example.sh examples/lock-app/nrf5 out/lock_app_debug
             - name: Build example nRF5 Lighting App
               run: scripts/examples/gn_nrf_example.sh examples/lighting-app/nrf5 out/lighting_app_debug
-            - name: Preserve artifacts
-              run: |
-                  mkdir -p example_binaries/nrf-build
-                  cp out/lock_app_debug/chip-nrf52840-lock-example \
-                     out/lighting_app_debug/chip-nrf52840-lighting-example \
-                     example_binaries/nrf-build/
+            - name: Binary artifact suffix
+              id: outsuffix
+              uses: haya14busa/action-cond@v1.0.0
+              with:
+                  cond: ${{ github.event.pull_request.number == '' }}
+                  if_true: "${{ github.sha }}"
+                  if_false: "pull-${{ github.event.pull_request.number }}"
+            - name: Uploading Binaries
+              uses: actions/upload-artifact@v2
+              with:
+                  name: ${{ env.BUILD_TYPE }}-example-build-${{ steps.outsuffix.outputs.value }}
```

Comment:
The binary upload names will be used for bloat check. Could we make the BUILD_TYPE be something like 'gn_nrf' and such so that bloat check compares the right versions?

Alternatively we can remove the uploading of binaries from the autotools build. The binary upload is for bloat purpose only anyway. If we go this latter path, there will be a time where gn builds will be bloat checked against the autotools versions.

Reply:
Changed to gn_nrf and gn_linux.